libraries/AR_WPNav/AR_WPNav.cpp

```diff
@@ -81,6 +82,15 @@ const AP_Param::GroupInfo AR_WPNav::var_info[] = {
     // @Increment: 0.1
     // @User: Standard
     AP_GROUPINFO("SPEED_MIN", 6, AR_WPNav, _speed_min, 0),
+    
+    // @Param: PIVOT_EXIT
```

Comment:
maybe PIVOT_TOL for pivot tolerance?

Reply:
I guess that makes sense...  Maybe @peterbarker and @rmackay9 can suggest something?